tests/scalers/redis/redis_cluster_streams_lag/redis_cluster_streams_lag_test.go

```diff
@@ -187,35 +187,38 @@ func TestScaler(t *testing.T) {
 	data, templates := getTemplateData()
 
 	CreateKubernetesResources(t, kc, testNamespace, data, templates)
-	testScaleOut(t, kc, data)
-	testScaleIn(t, kc)
-	DeleteKubernetesResources(t, testNamespace, data, templates)
+	t.Log("--- testing activation ---")
+	testActivationValue(t, kc, data, 3)
+
+	t.Log("--- testing scale out with one more than activation ---")
+	testScaleOut(t, kc, data, 1, 1)
+
+	t.Log("--- testing scale out with many messages ---")
+	testScaleOut(t, kc, data, 100, maxReplicaCount)
```

Comment:
You can merge both as once the activation has been tested, the scaling from 0->1 and 1->N can be done together, if 1->N works it means that 0->1 works indeed

```suggestion
	t.Log("--- testing scale out with many messages ---")
	testScaleOut(t, kc, data, 100, maxReplicaCount)
```

Reply:
I actually separated the two just to make sure that exactly one message over the activation value was all that was needed  to trigger the scaling. If I test 0-> N all at once, then I'm not sure that the scaling started exactly at the activation value, i.e. it may have started at 10 or 25 messages rather than the set value of 4.